sourcemash/api/feeds.py

```diff
@@ -10,6 +10,13 @@
 
 from sourcemash.models import Feed, UserItem, Item
 
+import os
+from rq import Queue
+from worker import create_worker
+from worker.scraper import scrape_feed_articles
+
+conn = create_worker(os.environ.get("APP_CONFIG_FILE") or "development")
```

Comment:
Okay, I'll look at that repo tomorrow then. Can't do it tonight. Fever coming back and dying here.